src/Form/CommitteeCommandType.php

```diff
@@ -41,6 +44,13 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'default_protocol' => null,
             ])
         ;
+
+        if (!$committee || $committee->isWaitingForApproval()) {
+            $builder->add('photo', FileType::class, [
```

Comment:
If a user submit this filed as empty, will it erase the photo?

Reply:
No, we have not remove the uploaded file only replace with new one.